blueprints/app/files/config/optional-features.json

```
@@ -2,5 +2,6 @@
   "application-template-wrapper": false,
   "default-async-observers": true,
   "jquery-integration": false,
-  "template-only-glimmer-components": true
+  "template-only-glimmer-components": true,
+  "no-implicit-route-model": false
```

```suggestion
  "no-implicit-route-model": true
```

I feel like we should be defaulting to the new (desired) behaviour when generating a new app with this blueprint 🤔 

I don't know if we have a policy for this written down for when we introduce a new optional feature, but considering we are hoping to remove this code in 6.0 it seems reasonable for new apps generated between now and then to be generated with the new stuff

---

I agree. I think the same was done for the Octane optional features, and we also do the same for the `EXTEND_PROTOTYPES` env var => https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/config/environment.js#L10

---

The defaults of the optional features in Octane were set to the "on" behavior only at Octane (the edition) being released. Prior to that they were the defaults. 

I agree with @mansona however, especially considering it targets 6.0, but that is not what the RFC says. @mansona could you ask this question on the RFC / propose a change to the text over there?